deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,93 @@
+
+
+elasticsearch:
+  host: ""
+  user: ""
+  pass: ""
+
+cloudDefend:
+  enabled: false
+
+kubernetes: 
+  version: 1.51.0
+  enabled: false
+  namespace: "default"
+      
+  system:
```

Comment:
Is not this the system integration? If yes should move to the same level as Kubernetes

Reply:
nope this `system` is this one https://docs.elastic.co/integrations/kubernetes/kubelet#system